### PR TITLE
gnome-browser-connector: Add at v42.1

### PR DESCRIPTION
**Pull request by ermo**
AFAICT, this only rundeps on python-gobject, which in turns depends on gobject and gobject-introspection libraries (and a python3 interpreter).

**Test Plan**

- Install gnome-browser-connector via moss
- Install GNOME Shell integration extension via Firefox
- Install GNOME Extensions app via flathub
- In FF, go to https://extensions.gnome.org/ and install WinTile-Beyond

### CODE REVIEW

File: g/gnome-browser-connector/stone.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#                
+name        : gnome-browser-connector
+version     : '42.1'
+release     : 1
+homepage    : https://github.com/GNOME/gnome-browser-connector
```
Comment (ikeycode) [inline g/gnome-browser-connector/stone.yaml:9]: wouldn't it be gitlab?

**Review by ikeycode — COMMENTED** [review]
for monitoring.yml i believe it's https://release-monitoring.org/project/311451/

File: g/gnome-browser-connector/stone.yaml (hunk 1 above)
Comment (ermo) [reply g/gnome-browser-connector/stone.yaml:9]: Derp.

**Review by ikeycode — APPROVED** [review]
lgtm!

**Pull request merged**